application/pr/viewer.jsx

```diff
@@ -43,9 +47,28 @@ export default class PRViewer extends React.Component {
                                         })
                                        ));
             })
-            .then(() => this.setState({groupDetails, status: "ready"}))
+            .then(() => {
+                return fetch(pp + "api/team-contact-of", { credentials: "include"})
+                    .then(utils.jsonHandler)
+                    .then((data) => {
+                        if (!data.hasOwnProperty('error')) {
+                            teamcontact = groupDetails.some((g) => {
+                                return data.map((wg) => { return wg.title; }).includes(g.name);
```

Comment:
likeiwse `=> { return` is redundant

Reply:
Good point on starting from `data`.
Regarding the name comparison, I didn't go with the ID because it's not directly available and we have to compare with the `href` which contains the ID but I guess that would be safer to rely on it.
All addressed in https://github.com/w3c/ash-nazg/pull/163/commits/3491667e19f54f1d4a3e43e40c05b151d66203a0